src/subscriptions/reaperLambda/subscriptionReaper.ts

```diff
@@ -3,28 +3,17 @@
  *  SPDX-License-Identifier: Apache-2.0
  *
  */
-import { DynamoDbDataService, DynamoDb } from 'fhir-works-on-aws-persistence-ddb';
+import { DynamoDbDataService } from 'fhir-works-on-aws-persistence-ddb';
 
-const enableMultiTenancy = process.env.ENABLE_MULTI_TENANCY === 'true';
-const dbServiceWithTenancy = new DynamoDbDataService(DynamoDb, false, {
-    enableMultiTenancy,
-});
-const dbService = new DynamoDbDataService(DynamoDb);
-
-const reaperHandler = async (event: any) => {
-    console.log('subscriptionReaper event', event);
+const reaperHandler = async (dbService: DynamoDbDataService, dbServiceWithTenancy: DynamoDbDataService) => {
     const subscriptions = await dbService.getActiveSubscriptions({});
     const currentTime = new Date();
     // filter out subscriptions without a defined end time.
     // check if subscription is past its end date (ISO format)
     // example format of subscriptions: https://www.hl7.org/fhir/subscription-example.json.html
-    console.log(subscriptions);
     return Promise.all(
         subscriptions
             .filter((s: Record<string, any>) => {
-                if (!s.end) {
-                    return false;
-                }
```

Comment:
why is this being removed? I though it was a bugfix

Reply:
Restored this bugfix, it got lost in the merge conflict resolution.